src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -0,0 +1,369 @@
+          <div className="p-text--small u-text--muted u-no-margin--bottom">
+            From: {item.source}
+          </div>
+        ),
+        override: isNoneDevice ? (
+          <Button
+            appearance="base"
+            type="button"
+            title="Reattach volume"
+            onClick={() => {
+              ensureEditMode(formik);
+              removeDevice(noneDeviceId, formik);
+            }}
+            className="has-icon u-no-margin--bottom"
+          >
+            <Icon name="connected"></Icon>
+            <span>Reattach</span>
+          </Button>
+        ) : (
+          <Button
+            appearance="base"
+            type="button"
+            onClick={() => {
+              ensureEditMode(formik);
+              addNoneDevice(item.key, formik);
+            }}
+            className="has-icon u-no-margin--bottom"
+            dense
+          >
+            <Icon name="disconnect"></Icon>
+            <span>Detach</span>
+          </Button>
+        ),
+      }),
+    );
+
+    Object.keys(item.proxy).forEach((key) => {
+      if (key === "name" || key === "type") {
+        return null;
+      }
+
+      inheritedRows.push(
+        getInheritedDeviceRow({
+          label: deviceKeyToLabel(key),
+          inheritValue: item.proxy[key as keyof typeof item.proxy],
+          readOnly: false,
+          isDeactivated: isNoneDevice,
+        }),
+      );
+    });
+  });
+
+  const customRows: MainTableRow[] = [];
+  formik.values.devices.forEach((formDevice, index) => {
+    if (formDevice.type !== "proxy") {
+      return;
+    }
+    const device = formik.values.devices[index] as LxdProxyDevice;
+
+    customRows.push(
+      getConfigurationRowBase({
+        className: "no-border-top custom-device-name",
+        configuration: (
+          <RenameDeviceInput
+            name={device.name}
+            index={index}
+            setName={(name) => {
+              ensureEditMode(formik);
+              void formik.setFieldValue(`devices.${index}.name`, name);
+            }}
+          />
+        ),
+        inherited: "",
+        override: (
+          <Button
+            className="u-no-margin--top u-no-margin--bottom"
+            onClick={() => {
+              ensureEditMode(formik);
+              removeDevice(index, formik);
+            }}
+            type="button"
+            appearance="base"
+            hasIcon
+            dense
+            title="Detach Proxy"
+          >
+            <Icon name="disconnect" />
+            <span>Detach</span>
+          </Button>
+        ),
+      }),
+    );
+    customRows.push(
+      getProxyDeviceFormRows("NAT mode", "nat", index, [
+        { label: "Enabled", value: "true" },
+        { label: "Disabled", value: "false" },
+      ]),
+    );
+    customRows.push(
+      getProxyDeviceFormRows("Bind", "bind", index, [
+        { label: "Host", value: "host" },
+        { label: "Instance", value: "instance" },
+      ]),
+    );
```

Comment:
I think this one should be first and with a help text according to the latest design.